supervisor/docker/__init__.py

```diff
@@ -230,3 +230,17 @@ def repair(self) -> None:
             _LOGGER.debug("Networks prune: %s", output)
         except docker.errors.APIError as err:
             _LOGGER.warning("Error for networks prune: %s", err)
+
+    def check_denylist_images(self) -> bool:
+        """Return a boolen if the host has images in the denylist."""
+        has_denied_images = False
+        for image in self.images.list():
+            for tag in image.tags:
+                imagename = tag.split(":")[0]
+                if imagename in DOCKER_IMAGE_DENYLIST:
+                    has_denied_images = True
+                    _LOGGER.error(
+                        "Found image %s, this is not supported, remove this from the host!",
+                        imagename,
+                    )
+        return has_denied_images
```

Comment:
This show for the same image multible times the error log. Let's write all found image into a set() and check if the set have an image inside, if no, return False otherwise show an error which include the list from the set.